.nuget/packages.config

```diff
@@ -2,4 +2,5 @@
 <packages>
   <package id="NuGet.CommandLine" version="2.8.2" />
   <package id="StyleCop.Analyzers" version="1.0.0-alpha003" />
+  <package id="StyleCop.Analyzers" version="1.0.0-alpha004" />
```

Comment:
:bulb: You can go ahead and remove this line. I'll correct this file in a separate commit.